fieldmaptrack/track.py

```diff
@@ -94,23 +96,33 @@ def calc_reference_point(self):
         b = 0.5*((x1+x2) - a * (z1+z2))
         return (b, 0.0)
 
+    def calc_field(self, rx, ry, rz):
+        """."""
```

Comment:
you could add a docstring here to make it clear the order of the components that will be returned.